webapp/channels/src/components/suggestion/at_mention_provider/at_mention_provider.tsx

```diff
@@ -458,3 +460,26 @@ export default class AtMentionProvider extends Provider {
         };
     }
 }
+
+defineMessages({
```

Comment:
I don't find where exactly these are used. It would be great if we could use this variables in place, not just defining them here. I understand it may require too much involvement and be considered out of the scope, but if that is the case, it would be great to leave a comment in the code here pointing where these should be used, so we can do it more straightforward the next time we run into this (maybe HelpWanted?).

Reply:
And the same for the other strings you moved to the other providers.

Reply:
The reason you can't find them is because the `SuggestionList` code is constructing them here: https://github.com/mattermost/mattermost/blob/master/webapp/channels/src/components/suggestion/suggestion_list.tsx#L208

I'd definitely like to change that since we've definitely had issues before with the dividers in that list not being translated correctly. I did actually try to change it so that the `Provider` passed a message descriptor for the divider, but I ended up running into a bunch of issues due to things like the quick switch modal and forward post modal which use (and arguably misuse) the suggestion code. The divider code has a lot of ugliness because I didn't originally expect most of the autocompletes to have dividers, and it's somewhere on my list of things to tackle

Reply:
It is not pretty (and more than happy to move it to a different PR) but I already did something similar for group roles (and not sure if any other): https://github.com/mattermost/mattermost/blob/master/webapp/channels/src/components/admin_console/permission_schemes_settings/strings/groups.tsx